crates/cubecl-wgpu/src/compute/server.rs

```diff
@@ -373,18 +364,7 @@ where
         self.queue.submit([encoder.finish()]);
 
         self.tasks_count = 0;
-        self.compute_storage_used.clear();
-
-        self.copy_handles_used.retain_mut(|x| {
-            // For some unknown reason, we have to make sure
-            // a buffer isn't used more than once not just in the current
-            // submission, but also in the next one.
-            //
-            // This really needs a better explanation of why this is, or
-            // some investigation, maybe it's a wgpu bug.
-            x.1 += 1;
-            x.1 < 2
-        });
+        self.storage_in_flight.clear();
```

Comment:
Why syncing means that we can remove the external readonly buffers from the list?